server.port=9001
spring.jpa.hibernate.ddl-auto=update
spring.cloud.gcp.logging.enabled=true

spring.cloud.jpa.properties.hibernate.format_sql=true
#spring.datasource.url=jdbc:mysql://35.200.164.231:root:3306/sys
#spring.cloud.datasource.url=jdbc:mysql://localhost:root:3306/sys
spring.cloud.gcp.sql.database-name=test
spring.cloud.gcp.sql.instance-connection-name=impactful-arbor-375511:asia-south1:root
# So app starts despite "table already exists" errors.
spring.datasource.continue-on-error=true
# Enforces database initialization
spring.sql.init.mode=always

# Leave empty for root, uncomment and fill out if you specified a user
spring.datasource.username=root

# Uncomment if root password is specified
spring.datasource.password=password

#spring.cloud.gcp.project-id=
#spring.cloud.gcp.credentials.location=file:/path/to/service-account.json
